### DISCUSSION TITLE
"addTransceiver" is supported in Opera browser but it is listed under unsupported in docs

### DISCUSSION THREAD

**Discussion by Abhishekabi** [mdn/mdn-community#330] [Browser compatibility data] [answered]
Here the [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/addTransceiver#browser_compatibility) is showing wrong data for opera. Please refer the below screenshots.
<img width="1478" alt="Screenshot 2023-02-17 at 5 22 36 PM" src="https://user-images.githubusercontent.com/27338062/219647071-2fe6dd74-415b-4ed5-b69a-2a5770a60625.png">

**Comment by schalkneethling** [comment, marked as answer]
Thank you for letting us know, @Abhishekabi. This would probably be better to file on the BCD repository (https://github.com/mdn/browser-compat-data), but let's see what @queengooborg thinks.

**Reply by queengooborg** [reply]
Agreed, this is something that should be filed as an issue in the repository itself!

**Reply by Abhishekabi** [reply]
Filed as an issue in BCD repo, thanks team!

**Comment by pepelsbey** [comment]
It seems like the bug has been filed, and this discussion is no longer relevant. Closing as resolved. Thanks!
